demo/src/main/java/dev/transformerkt/demo/transformer/TransformerRepo.kt

```diff
@@ -124,7 +127,17 @@ class TransformerRepo @Inject constructor(
                     val inputChannels = settings.audioOverlay.uri.audioTracks(context)
                     effects {
                         audioSampleRate(44100)
-                        volume(settings.audioOverlay.volume, inputChannels = inputChannels)
+                        if (settings.audioOverlay.fade) {
+                            fadeAudioOut(
+                                totalDurationUs = duration,
+                                inputChannels = inputChannels,
+                                initialVolume = settings.audioOverlay.volume,
+                                finalVolume = 0.3f,
```

Comment:
This should use a value from settings instead of being hard-coded. Looks like an oversight

Reply:
This is just the demo, and the final volume is set to 0.3f so that it doesn't go to zero.

Reply:
Oh duh thanks!